nni/nas/space/frozen.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+__all__ = ['current_model', 'model_context']
+
+import copy
+from typing import Optional, Dict, Any
+from nni.mutable import frozen_context, ContextStack
+
+
+def current_model() -> Optional[dict]:
+    """Get the current architecture dict in :func:`model_context`.
+
+    This is only valid when called inside :func:`model_context`.
+    By default, only when :class:`~nni.nas.space.SimplifiedModelSpace` is used as the model format,
+    :func:`current_model` is meaningful.
+
+    Returns
+    -------
+    Architecture dict before freezing, produced by strategy.
+    If not called inside :func:`model_context`, returns None.
```

Comment:
Why not throw?

Reply:
We'll see in `nas.nn` that missing context is a rather normal case and shouldn't throw.